src/components/Settings/SettingsNewAddressbook.vue

```diff
@@ -47,16 +44,10 @@ export default {
 	directives: {
 		clickOutside
 	},
-	// props: {
-	// 	addressbooks: {
-	// 		type: Array,
-	// 		default() {
-	// 			return {}
-	// 		}
-	// 	}
-	// },
 	data() {
 		return {
+			// match any expression
+			addressBookRegex: new RegExp('/.*/')
```

Comment:
:+1:

Reply:
Hmm this does not seem to be working anymore 😞. 
It works on here: https://regex101.com/r/cO8lqs/2
I need to look at it some more.

Reply:
Remove the regex from the input then and add a `TODO:` stating we need to add a check later :)